### ISSUE TITLE
[CoE Starter Kit - Feature]: Upgrade for Flow Creator information in Flow table

### ISSUE THREAD

**Issue by annsarP** [microsoft/coe-starter-kit#5905]
### Is your feature request related to a problem? Please describe.

Hello!
We are monitoring status of flows in our tenant and if they are pending suspension. Power Platform Admin View app and Admin Sync flows are really helpful with that. However, sometimes we need to reach out to owners to inform them. The Flow Creator UPN column gives us only UPN, and there are cases that DerivedOwner or Maker columns are not containing creator. (We are focusing on creators so much because flows are relying on their license so they are the ones that we need to get in touch with or assign a license.)

### Describe the solution you'd like

Would it be possible to add new lookup / e-mail column in Flow table presented in Power Platform Admin View to store more Flow Creator information?

### Describe alternatives you've considered

We created a flow that is creating and sending weekly report to admins with list of all suspended and pending suspension flows but we were missing Flow Creator name or their e-mail address. 
We added additional action in SYNC HELPER - Cloud Flows to get Flow Creator profile. 
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/f69bd719-f7a6-4649-b35c-052968255254)
Now we only need a column we can put the value Mail in. :)

### Additional context?

Flow Creator UPN column is an e-mail type column, so after clicking it it tries to send an e-mail to UPN. 
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/dee08fb9-b3ee-40a6-afe1-154cd160e396)
Is that expected setup and behavior?

**Comment by Jenefer-Monroe** [comment]
We actually wish we could reduce these duplicates so would not want to add more. 
But you can do an expand query on the flow table to get this information. 

Here is an example from the Governance solution of that. You would just change the column that is selected 
![Image](https://github.com/microsoft/coe-starter-kit/assets/89584709/a2b8b150-08a8-4231-ba2d-25e1255914b7)

Let me know if that makes sense.

**Comment by annsarP** [comment]
Hello!
Thank you for this solution. I tested it but the real creator is still missing from the output. 
Flow Creator UPN column (admin_flowcreatorupn) value is not matching  _admin_flowcreator_value@OData.Community.Display.V1.FormattedValue (and any other value either). It's giving different user. 

If you would like to avoid adding new column then I am closing this case.
I created custom table for that, and I will be putting gathered flows in there (with display name of the flow creator), then based on this list I will be sending a report. 

Thank you for your response and time! 😊
